widget/editor.go

```diff
@@ -936,6 +1171,153 @@ func (e *Editor) NumLines() int {
 	return len(e.lines)
 }
 
+func (e *Editor) IsSelected() bool {
```

Comment:
Do you need IsSelected externally? If not, I'd lean towards deleting it entirely, or perhaps replace it with SelectionLen(gth).